src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -605,6 +662,11 @@ public String getQuestionResultStatisticsCsv(
                 String percentageFrequency = df.format(rubricStats[i][j] * 100) + "%";
                 csv.append("," + percentageFrequency + " (" + responseFrequency[i][j] + ")");
             }
+
+            if (hasAssignedWeights) {
+                csv.append("," + dfAverage.format(rubricStats[i][numOfRubricChoices]));
```

Comment:
`rubricStats` stores the result of `calculateRubricStats`, an array containing the percentage frequencies and averages.